### ISSUE TITLE
Not running

### ISSUE THREAD

**Issue by ahunter87** [crooke/ignition-translation-converter#1]
Hi,
 I'm very new to python and I'm trying to get this tool to work but cant seem to get it to accept my arguments. Any help would be great! 

running the program as is give a few errors back
Thanks

**Comment by crooke** [comment]
Can you post the error message? My first thoughts would be to make sure you're using Python 3 and have the lxml python package installed.

**Comment by ahunter87** [comment]
Sorry for the delay in response, I did need to install lxml package. Now that the package is install, I am getting the message: 

usage: main.py [-h] input_file output_file
main.py: error: the following arguments are required: input_file, output_file
[Finished in 0.7s]

I'm currently not sure how to input the arguments

**Comment by ahunter87** [comment]
ok for the most part I got it running. The issues I was having was coming from the python version I was running, not having lxml package installed, as well as not having Argparse installed. I had to down grade from python 3.7 to 3.4 the highest version I could find that had both those lxml and argparse packages...................................................................
 I'm still having a issue the python program runs the converter from XML to CSV and says completed  in Cmd . The program creates a new file CSV but it only has two words in it  " Term Translation".
The original xml file has 1,500 tags in it.

Just want to say Thank you ahead of time for any help!

![conerter1](https://user-images.githubusercontent.com/36282409/36450281-45b5287a-1685-11e8-9e98-4f0e2e18ae7c.jpg)
